EventListeners.js

```diff
@@ -53,10 +53,21 @@ export function setupEventListener(logic) {
             // )
         }
     })
+
+    // listen for when restart is clicked is clicked
+    const restartBtn = document.querySelector('.restartBtn')
+
+    restartBtn.addEventListener('click', () => {
+        logic.mutate(mutatorFns.resetBasketValue)
```

Comment:
As you're calling 4 different mutations here, it might make sense to create a new mutation - maybe `resetGame` - which takes care of ensuring all 4 of these mutations occur when it gets called.

Not super important, but I'd probably do it like this to hide away more of the specifics from the event handlers. Mainly just want to give you another perspective, but not implying this is wrong